SimPEG/PF/Magnetics.py

```diff
@@ -23,361 +26,476 @@ class MagneticIntegral(Problem.LinearProblem):
     forwardOnly = False  # If false, matrix is store to memory (watch your RAM)
     actInd = None  #: Active cell indices provided
     M = None  #: Magnetization matrix provided, otherwise all induced
-    rtype = 'tmi'  #: Receiver type either "tmi" | "xyz"
+    rxType = 'tmi'  #: Receiver type either "tmi" | "xyz"
```

Comment:
same comments here on naming conventions and docstrings (as much as possible, stick with lower case and underscores, no uppercase for property names)